.codecov.yml

```diff
@@ -1 +1,10 @@
 comment: false
+coverage:
+  status:
+    project:
+      default:
+        target: 85
```

Comment:
Added checkbox to https://github.com/iterative/dvc/issues/4468 to address it later.